shared/chat/conversation/header-area/index.native.tsx

```diff
@@ -37,7 +37,8 @@ const ShhIcon = React.memo(function ShhIcon() {
 const useMaxWidthStyle = () => {
   const {width} = useWindowDimensions()
   const hasBadge = useBackBadge() > 0
-  return React.useMemo(() => ({maxWidth: width - 140 - (hasBadge ? 40 : 0)}), [width, hasBadge])
+  const w = width - 140 - (hasBadge ? 40 : 0)
```

Comment:
when starting up already on this view when we init it has some race, so just set the size